examples/User Creation & Provisioning/README.md

```diff
@@ -65,7 +103,7 @@ Run the script:
 ./Users_Create_Provision.ps1
 ```
 
-You will see following output to confirm the script started:
+If there is no parameters still missing, you will see following output to confirm the script started:
```

Comment:
When all parameters are defined, you will see following output to confirm the script started: